src/main/kotlin/frc/robot/RobotStateMachine.kt

```diff
@@ -34,11 +34,11 @@ enum class IntakeState(val innerPercent: Double, val outerPercent: Double) {
 }
 
 //this represents the DESIRED trunk statautoshooe
-enum class TrunkPosition(val angle: Double, val position: Double) {
+enum class TrunkPose(val angle: Double, val position: Double) {
```

Comment:
Why did you rename this? The new name isn't more descriptive, and it just clogs up the changes

Reply:
calling it position is confusing because it contains more than position